### PR TITLE
Added proper about page with tools versions

### PR DESCRIPTION
**Pull request by FrenchBen**
Will probably need some better banner graphic and design input from @Elesant
This will help provide an homogenous 'About' page for all kitematic platforms.

Preview:
kitematic-version

Signed-off-by: FrenchBen <me@frenchben.com>

### CODE REVIEW

File: src/menutemplate.js [hunk 1]
```diff
@@ -17,7 +17,13 @@ var MenuTemplate = function () {
       submenu: [
       {
         label: 'About Kitematic',
-        selector: 'orderFrontStandardAboutPanel:'
```
Comment (timfallmk) [inline src/menutemplate.js]: I think we need a switch here. The `selector` is necessary for OSX.

Comment (FrenchBen) [inline src/menutemplate.js]: Not necessary, as it's commented out :P - I should delete it completely to prevent any further confusion.

Comment (timfallmk) [inline src/menutemplate.js]: Sorry, I meant it _is_ necessary for OS X. The switch would be to check with arch you're on. ;)

**Comment by cclauss** [comment]
This looks great!  Really useful.

**Comment by FrenchBen** [comment]
Latest changes include @Elesant's designs
about-cartoon

**Comment by timfallmk** [comment]
:+1:

**Comment by Elesant** [comment]
This is awesome! Let's merge it :)

**Comment by cclauss** [comment]
Beautiful!!

**Pull request merged**